### ISSUE TITLE
Props actionButtonsPosition

### ISSUE THREAD

**Issue by suplere** [mesqueeb/blitzar#86]
Hello,

I use BlitzForm (version 1.0.3), but props actionButtonsPosition does not any effect, Actions buttons are allways on the top-right possition.

**Comment by mesqueeb** [comment]
@suplere I will test soon!! sorry haven't found time to look into it yet.

PS: you can also help if you want with a PR

`git checkout v1.0.0`
`npm i`
`npm run dev`

should be good to develop this. I have checked the docs and we currently have no example section on the `actionButtonsPosition` at all, so feel free to add it and test the implementation.

**Comment by mesqueeb** [comment]
@suplere i finally released the Vue 3 version officially

- See new docs:
https://blitzar.cycraft.co/


I will look into this issue soon

**Comment by ErikGoH** [comment]
Hey I also noticed the same behavior when trying to use `action-buttons-position="bottom"` I did some digging and I think I have found what's happening here.

# Problem
In short: The style that adds the flex display to `.blitz-form` is only doing it when `.q-form` is also present by this I mean in `packages/form/src/BlitzForm.vue` we have the following css

```css
<style lang="sass">
.blitz-form
  &.q-form
    display: flex
```

I found about the missing `display: flex` by looking at the devtools css 
![image](https://user-images.githubusercontent.com/58312976/150631351-7c3c2654-ddf2-4167-8a8c-6d78d78e2e69.png)
I quickly added the `display: flex` to the element and the actionButtons moved correctly to the bottom.
# Possible Fix
I don't know if we could just always add the `display: flex` to the `.blitz-form` class

```diff
<style lang="sass">
.blitz-form
  display: flex
-  &.q-form
-    display: flex
```

I tried following the steps mentioned above to try a make a PR but I hit a issue of having two different vue versions between the blitzar package and the project I am using blitzar, then I tried upgrading my project vue version and everything just broke (got a bunch of type errors) :face_exhaling: , so I could not test it to see if that would work and that it would not break any other style.

I think I am going to need to create another simple project without quasar to easily test the package. 

# Possible workarounds

## Using quasar framework
You could pass to the `form-component` prop the `q-form` component so blitzar uses that instead of the default div, remember to register `q-form` in the `quasar.conf.js` 

```html
 <blitz-form
    :schema="schema"
    form-component="q-form"
    :action-buttons="['save']"
    :action-button-defaults="{save: actionButtonSave}"
    action-buttons-position="bottom"
    mode="edit"
  />
```
```js
// quasar.conf.js
...
// https://quasar.dev/quasar-cli/quasar-conf-js#Property%3A-framework
  framework: {
    config: {},

    // iconSet: 'material-icons', // Quasar icon set
    // lang: 'en-US', // Quasar language pack

    // For special cases outside of where the auto-import strategy can have an impact
    // (like functional components as one of the examples),
    // you can manually specify Quasar components/directives to be available everywhere:
    //
    components: [
      'QInput',
      'QBtn',
      'QFile',
      'QIcon',
      'QForm', // <--------- registers the QForm component globally
    ],
...
```

## Using another framework
You could assign a class to the `blitzform` element and there add the `display: flex`

```html
<template>
  <blitz-form
    :schema="
    class="blitzform-display"
    :action-buttons="['save']"
    :action-button-defaults="{save: actionButtonSave}"
    action-buttons-position="bottom"
    mode="edit"
  />
</template>

<script setup lang="ts">
import 'blitzar/dist/style.css';
// import { BlitzForm } from '../../../../../blitzar/packages/form/dist/index.es';
import { BlitzForm } from 'blitzar';

import { schema, actionButtonSave } from './index';

</script>

<style scoped lang="sass">
.blitzform-display
  display: flex
</style>

```

**Comment by mesqueeb** [comment]
Fix made on latest version! `1.1.2` 

PS: @erikgoh thanks for looking into it. I was able to quickly fix it because of you!

<sub>--<br />Blitzar was made with 💜 by Luca Ban.<br />You cannot sponsor every project, but next time you do, think of <a href="https://github.com/sponsors/mesqueeb">this one</a> for its prolonged maintenance.</sub>

**Comment by mesqueeb** [comment]
just quick note, I had broken something about the BlitzTable with latest release.
fixed now!
